docs/contributors/release.md

```diff
@@ -253,6 +253,128 @@ Now, the branch is ready to be used to publish the npm packages.
 
 Now, the npm packages should be ready and a patch can be created and committed into the corresponding WordPress SVN branch.
 
+
+### Standalone Package Releases 
+
+The following workflow is needed when packages require bug fixes or security releases to be published to _npm_ outside of a WordPress release cycle.
+
+Note: Both the `master` and `wp/trunk` branches are restricted and can only be _pushed_ to by the Gutenberg Core team.
+
+Identify the commit hashes from the pull requests that need to be ported from the repo `master` branch to `wp/trunk`
+
+The `wp/trunk` branch now needs to be prepared to release and publish the packages to _npm_.
+
+Open a terminal and perform the following steps:
+1. `git checkout master`
+2. `git pull`
+3. `git checkout wp/trunk`
+4. `git pull`
+
+Before porting commits check that the `wp/trunk` branch does not have any outstanding packages waiting to be published:
+1. `git checkout wp/trunk`
+2. `npm run publish:check`
+
+Now _cherry-pick_ the commits from `master` to `wp/trunk`, use `-m 1 commithash` if the commit was a pull request merge commit:
+1. `git cherry-pick -m 1 cb150a2`
+2. `git push` 
```

Comment:
I personally push only specific branches, is it always safe to use `git push` with the current branch?

Reply:
Only @WordPress/gutenberg-core can push to either of those branches directly I believe, so some care should be taken, and it's not ideal

This entire workflow outlined could be refactored to create branches off of `wp/trunk` then create PRs back against that branch, probably worth doing in a follow up PR

Reply:
Yes, branching could work, although it's easier to apply commits one by one directly in the `wp/trunk` branch. We do it the same way in `master` when backporting publish and changelog changes.

FYI, I started automation around npm publish workflow when syncing plugin with `wp/trunk`:
https://github.com/WordPress/gutenberg/blob/431f76c1b32e12322d44c66fddc289431f7d6c8d/bin/commander.js#L704-L730

It's still in progress and needs more work though.